packages/features/bookings/lib/handleNewBooking/test/reschedule.test.ts

```diff
@@ -2193,7 +2531,7 @@ describe("handleNewBooking", () => {
           });
 
           const roundRobinHost2 = getOrganizer({
-            name: "RR Host 2",
```

Comment:
I made this change since [this](https://github.com/calcom/cal.com/blob/main/packages/lib/defaultEvents.ts) regex considers this name as three user names due to spaces b/w them.